hbshelpers/generateTransformFacetsCall.js

```diff
@@ -0,0 +1,16 @@
+const TransformFacetsCall = require('./codegen/TransformFacetsCall');
+
+/**
+ * Generates a transformFacets function based on the Facets 'fields' config option
+ * 
+ * @param {Object} context 
+ * @returns {string}
+ */
+module.exports = function generateTransformFacetsCall(config) {
```

Comment:
This code generation is super cool! but before jumping off the deep end, is there any way we can achieve this with normal handlebars partials, or even just slapping a new method in HitchhikerJS?